From 2fc5caafb1e9fe57be8ca4f4ca8fd047f3bf6b41 Mon Sep 17 00:00:00 2001
From: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Date: Thu, 23 Jan 2020 19:51:10 +0530
Subject: [PATCH 1019/1851] usb: dwc3: core: use first_trb_index instead of
 start_slot

commit 08ea9c5e80f25c69cd69be195bcab685d9b541f2 from
https://github.com/Xilinx/linux-xlnx.git

start_slot is now renamed to first_trb_index. This patch
adds first_trb_index entry into dwc3_request which is
used by core.c

Signed-off-by: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Piyush Mehta <piyush.mehta@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/usb/dwc3/core.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/dwc3/core.h b/drivers/usb/dwc3/core.h
index fd0805778e44..e938a37250be 100644
--- a/drivers/usb/dwc3/core.h
+++ b/drivers/usb/dwc3/core.h
@@ -902,6 +902,7 @@ struct dwc3_request {
 
 	unsigned		num_pending_sgs;
 	unsigned int		num_queued_sgs;
+	u8			first_trb_index;
 	unsigned		remaining;
 
 	unsigned int		status;
-- 
2.31.1

